examples/openshift-lightspeed-tls.yaml

```diff
@@ -100,6 +100,16 @@ spec:
             mountPath: /app-root/config/openai
           - name: tls-certs
             mountPath: /app-root/certs
+      - name: user-data-collector-sidecar
+        image: quay.io/openshift/lightspeed-service-api-udc:latest
+        # TODO: launch command
+        env:
+          - name: OLS_USER_DATA_PATH
```

Comment:
should be able to load/read the same config.yaml that the main ols container is using.